docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,1903 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    import chromedriver_binary
</pre><p>Wow, this is sort of terrifying, but eh on the other hand ... ship it!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Yeah, some black magic going on there. Don't look too closely ;)</p>